config/crd/bases/infrastructure.cluster.x-k8s.io_openstackclusters.yaml

```diff
@@ -13,481 +13,977 @@ spec:
-                security groups, one allowing SSH and API access from everywhere,
-                and another one that allows all traffic to/from machines belonging
-                to that group. In the future, we could make this more flexible.
-              type: boolean
-            network:
-              description: If NodeCIDR cannot be set this can be used to detect an
-                existing network.
-              properties:
-                adminStateUp:
-                  type: boolean
-                description:
-                  type: string
-                id:
-                  type: string
-                limit:
-                  type: integer
-                marker:
-                  type: string
-                name:
-                  type: string
-                notTags:
-                  type: string
-                notTagsAny:
-                  type: string
-                projectId:
-                  type: string
-                shared:
-                  type: boolean
-                sortDir:
-                  type: string
-                sortKey:
-                  type: string
-                status:
-                  type: string
-                tags:
-                  type: string
-                tagsAny:
-                  type: string
-                tenantId:
-                  type: string
-              type: object
-            nodeCidr:
-              description: NodeCIDR is the OpenStack Subnet to be created. Cluster
-                actuator will create a network, a subnet with NodeCIDR, and a router
-                connected to this subnet. If you leave this empty, no network will
-                be created.
-              type: string
-            saKeyPair:
-              description: SAKeyPair is the service account key pair.
-              properties:
-                cert:
-                  description: base64 encoded cert and key
-                  format: byte
-                  type: string
-                key:
-                  format: byte
-                  type: string
-              type: object
-            subnet:
-              description: If NodeCIDR cannot be set this can be used to detect an
-                existing subnet.
-              properties:
-                cidr:
-                  type: string
-                description:
-                  type: string
-                enableDhcp:
-                  type: boolean
-                gateway_ip:
-                  type: string
-                id:
-                  type: string
-                ipVersion:
-                  type: integer
-                ipv6AddressMode:
-                  type: string
-                ipv6RaMode:
-                  type: string
-                limit:
+              ready:
+                type: boolean
+            required:
+            - ready
+            type: object
+        type: object
+    served: true
+    storage: true
```

Comment:
storage of v1alpha2 should be false. I got the following error.
```
Error from server (Invalid): error when creating "infrastructure-components.yaml": CustomResourceDefinition.apiextensions.k8s.io "openstackclusters.infrastructure.cluster.x-k8s.io" is invalid: [spec.versions: Invalid value: []apiextensions.CustomResourceDefinitionVersion{apiextensions.CustomResourceDefinitionVersion{Name:"v1alpha2", Served:true, Storage:true, Schema:(*apiextensions.CustomResourceValidation)(0xc00ba6d1d0), Subresources:(*apiextensions.CustomResourceSubresources)(nil), AdditionalPrinterColumns:[]apiextensions.CustomResourceColumnDefinition(nil)}, apiextensions.CustomResourceDefinitionVersion{Name:"v1alpha3", Served:true, Storage:true, Schema:(*apiextensions.CustomResourceValidation)(0xc00ba6d1d8), Subresources:(*apiextensions.CustomResourceSubresources)(nil), AdditionalPrinterColumns:[]apiextensions.CustomResourceColumnDefinition(nil)}}: must have exactly one version marked as storage version, status.storedVersions: Invalid value: apiextensions.CustomResourceDefinitionVersion{Name:"v1alpha3", Served:true, Storage:true, Schema:(*apiextensions.CustomResourceValidation)(0xc00ba6d1d8), Subresources:(*apiextensions.CustomResourceSubresources)(nil), AdditionalPrinterColumns:[]apiextensions.CustomResourceColumnDefinition(nil)}: must have the storage version v1alpha3]
```

Reply:
the problem is the conflict when create alpha2 and alpha3 ,I think it's something not related to storage?